force-app/main/default/objects/Lead/validationRules/EmailorPhone.validationRule-meta.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>EmailorPhone</fullName>
+    <active>false</active>
+    <errorConditionFormula>OR(IF( ISPICKVAL(  Country__c  , &quot;United States&quot;) , IF( LEN( Phone )=10 , false, true)  , false),OR(ISBLANK(Email),ISBLANK(Phone)))</errorConditionFormula>
```

Comment:
This validation is not working properly check the conditions ensure the condition is achieving the task